index.d.ts

```diff
@@ -4,8 +4,12 @@ import { Component } from 'react';
 
 export interface WebViewProps extends React.HTMLAttributes<Electron.WebViewElement>, React.ClassAttributes<WebView> {
   src: string
-  
-  autosize?: boolean
+
+  autosize?: string;
```

Comment:
Yeah, but it says that for all of the boolean attributes like `nodeintegration` and `plugins`, so I'm fairly certain it behaves like a boolean where the actual value doesn't matter. It just matters if the attribute is there or not.